UM/Settings/ContainerRegistry.py

```diff
@@ -146,6 +148,13 @@ def findContainers(self, container_type = None, **kwargs):
                             continue
                         except AttributeError:
                             pass
+                    if key == "read_only":
+                        try:
+                            if value != container.isReadOnly():
+                                matches_container = False
+                            continue
+                        except AttributeError:
+                            pass
```

Comment:
I'm starting to think that instead of using `continue` everywhere, we should just use `elif`s.